data/language/en-GB.txt

```diff
@@ -4189,6 +4189,17 @@ STR_5877    :Software
 STR_5878    :Software (hardware display)
 STR_5879    :OpenGL
 
```

Comment:
You can remove this spacing and I think you've added some extra indents (2 tabs instead of 1)